### DISCUSSION TITLE
Resize entire cache without restarting?

### DISCUSSION THREAD

**Discussion by dsaingre** [facebook/CacheLib#325] [Q&A]
Hi,

I'm using cachelib to create a caching service. For the moment, my use of cachelib is quite simple and quite similar to what's adviced in the documentation ("Getting started with Cache library"). I would like to be able to dynamically resize my entire cache (and so triggering eviction if reducing cache size under current usage) without deleting and restarting the whole cache. In particular, I would like to be able to do it manually, controlling the expected cache size myself. 

Is there any way to do so using cachelib?

I've seen that I can shrink and grow individual Memory pools but it guess it only work in the cache total size limit...

Thanks a lot in advance !

**Comment by pbhandar2** [comment]
Yes you can resize pools but not the entire cache on the fly. 

Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
